src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+body {
+  margin: 0;
+  font-family: "Arial",
+    sans-serif;
+  font-size: $font-size;
+  min-height: 100vh;
+  @include flex-justify-align(center, center);
+}
+
+.main {
+  .calendar {
+    @for $i from 1 through $days-in-month {
+      &__day:nth-child(n+#{$i}):before {
+        content: "#{$i}";
+      }
+    }
+
+    @include flex-justify-align(flex-start, center);
+    flex-wrap: wrap;
+    gap: $gap;
+    max-width: $max-width;
+
+    &__day {
+      width: $cell-size;
+      height: $cell-size;
+      background-color: #eee;
+      border: $gap solid black;
+      box-sizing: border-box;
+      @include flex-justify-align(center, center);
+      cursor: pointer;
+
+      &:hover {
+        background-color: pink;
+        transform: translateY(-$cell-size/5);
+        transition: all 0.5s;
```

Comment:
transition should be set for all states not only hover
because not styles are removed immediately